api/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.conf.urls import url
+from api import views
+
+urlpatterns = [
+    url(r'^$', views.course_handler, name='course_handler'),
```

Comment:
Can you prefix these with `v1`. Also, it's a good idea to have a sub path for this simple file manipulation (if we require more, we can add different paths easily). Thus, the full path would be `/api/v1/files/` for example. Last part could be `files` or `upload` for example.